src/client.js

```diff
@@ -61,7 +61,8 @@ class Client extends Connection {
     try {
       const mcData = require('minecraft-data')('bedrock_' + this.options.version)
       this.features = {
-        compressorInHeader: mcData.supportFeature('compressorInPacketHeader')
+        compressorInHeader: mcData.supportFeature('compressorInPacketHeader'),
+        itemRegistryPacket: mcData.supportFeature('itemRegistryPacket')
```

Comment:
Both the client and servers have a .features but this was only added to the client. I updated the implementation for this so the feature check here isn't really needed anymore but we can keep feature here. A footgun however is that a undefined feature lookup here will not cause an exception and instead return false.